ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/glassfish/soteria/main/module.xml

```diff
@@ -28,6 +28,7 @@
     </properties>
 
     <resources>
+        <artifact name="${org.glassfish.soteria:soteria.spi.bean.decorator.weld}"/>
         <artifact name="${org.glassfish.soteria:jakarta.security.enterprise}"/>
```

Comment:
I would keep these ordered alphabetically.

Reply:
@rhusar The reason I put this first was to ensure this decorator was picked up first.